service/routes.py

```diff
@@ -181,32 +192,47 @@ def delete_shopcart(customer_id):
 # -----------------------------------------------------------
 
 
-@ app.route("/shopcarts/<customer_id>/<item_id>", methods=["POST"])
-def add_item(customer_id, item_id):
+@ app.route("/shopcarts/<customer_id>/items", methods=["POST"])
```

Comment:
If it just posts items, I think it doesn't need to do anything in the model.

Reply:
I think it just needs to check whether the shop cart exists.

Reply:
i think there is already a ticket for this right? ticket #55

Reply:
Right.  I means when call post /shopcarts/customer_id/items. We need to check if the shopcart is exist. If exist, then success, if not return error.

Reply:
I think this should be POST /shopcarts/<customer_id>/items/<item_id> ? While POST /shopcarts/<customer_id>/items should just checkwhether the shopcart exists?

Reply:
but shouldnt the `POST /shopcarts/<customer_id>/items/<item_id>` have the item info in the request body passed as JSON? so then I wouldnt have to pass item_id as a parameter? 
so i thought to add a new item to the customer's shopcart, i call `POST /shopcarts/<customer_id>/items`